content/docs/api-reference/artifacts_show.md

```diff
@@ -0,0 +1,102 @@
+# dvc.api.artifacts_show()
+
+Get the path and Git revision for an artifact tracked in a <abbr>DVC
```

Comment:
can artifact here be a link to something? (to give ppl context) or an `abbr`

Reply:
I think it might be out of scope of this PR, but we can make a follow up (I can take it) to add a glossary item and add more background to `cmd-ref/artifacts/index` to explain what is an `artifact`.

Reply:
Opened https://github.com/iterative/dvc.org/pull/4844

Reply:
@pmrowla Could you update the PR to use `<abbr>artifact</abbr>` and `<abbr>model registry</abbr>`?